.yarnrc.yml

```diff
@@ -4,8 +4,8 @@ enableGlobalCache: false
 
 nodeLinker: node-modules
 
-npmAuthToken: "${NPM_TOKEN}"
+npmAuthToken: '${NPM_TOKEN}'
 
-npmRegistryServer: "https://registry.npmjs.org/"
+npmRegistryServer: 'https://registry.npmjs.org/'
```

Comment:
**question:** have these been applied automatically by one of the package scripts or was this your IDE formatting this file based on different rules than the repo's prettier config?

Reply:
ah yes, I just had an experiment on my IDE config at that time 🤦 , thanks for noticing 👍

Reply:
but seems like its the correct one since the defined rules is

```
 quotes: ['error', 'single', { avoidEscape: true, allowTemplateLiterals: false }], // .eslintrc.cjs
  // and
 "singleQuote": true, // .prettierrc.cjs
```

probably this file should be ignored 🤔